### ISSUE TITLE
Accessibility of Visibility and Sharing With Section When Editing a Fileset

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/scholarsphere-3#1142]
When I select to edit a fileset that is part of a work and select the permissions tab, the "Visibility" section doesn't appear to be aligned with the "Visibility" section of the Work form and has missing and multiple form labels. Also the table with person or groups that you are sharing with has orphaned form labels.

**Comment by mtribone** [comment]
![screen shot 2018-01-10 at 10 10 10 am](https://user-images.githubusercontent.com/4163828/34779546-89165e70-f5ee-11e7-98d7-27cecd3146a9.png)

**Comment by mtribone** [comment]
![screen shot 2018-01-10 at 10 10 27 am](https://user-images.githubusercontent.com/4163828/34779553-904436cc-f5ee-11e7-87bf-e3a9d40a8b5c.png)

**Comment by mtribone** [comment]
And a couple contrast errors

![screen shot 2018-01-10 at 10 11 56 am](https://user-images.githubusercontent.com/4163828/34779624-c7e14f98-f5ee-11e7-884e-31da078cdf40.png)

**Comment by carolyncole** [comment]
We should update the file set form to use the work for instead of it' own form.  Then retest for accessibility.
